.github/workflows/agw-docker-load-test.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: AGW docker load test
+
+on:  # yamllint disable-line rule:truthy
+  workflow_run:
+    workflows:
+      - build-all
+    branches:
+      - master
+      - v1.*
+    types:
+      - completed
+
+jobs:
+  docker-load-test:
+    name: agw docker load tests
+    runs-on: ubuntu-latest
+    env:
+      SHA: "${{ github.event.workflow_run.head_commit.id }}"
+      MAGMA_ROOT: "${{ github.workspace }}"
+      CODE_DIR: "${{ github.workspace }}/experimental/cloudstrapper"
+      VARS_DIR: "${{ github.workspace }}/experimental/cloudstrapper/playbooks/roles/vars"
+      WORK_DIR: "${{ github.workspace }}/experimental/cloudstrapper/playbooks"
+      AGW_DOCKER_AMI: "ami-0087432b81fb36a56"
```

Comment:
We would need a plan to make this customizable. Maybe look for a name tag?

Reply:
yes we have similar issue with dev container and bazel, I would like to find something that could fit both use cases.

Reply:
https://github.com/magma/magma/issues/11113